docs/DeveloperGuide.md

```diff
@@ -470,6 +470,41 @@ The following sequence diagram shows how the MarkCommand operation works:
     * Pros: More comprehensive search for projects with the required keyword.
     * Cons: Much harder to implement, as it requires a field-less search.
 
+
+### \[Proposed\] `Sort` Feature
```

Comment:
This is not a `Proposed` feature, so you might want to remove the word.

Reply:
Changed!